interface/ceed-vector.c

```diff
@@ -940,27 +940,46 @@ int CeedVectorReciprocal(CeedVector vec) {
 /**
   @brief View a CeedVector
 
-  @param[in] vec    CeedVector to view
-  @param[in] fp_fmt Printing format
-  @param[in] stream Filestream to write to
+  @param[in] vec         CeedVector to view
+  @param[in] num_entries Number of CeedVector entries to view
```

Comment:
Maybe `CeedVectorViewRange()` taking a range (start, end, step)?

Reply:
I like this a lot. 
I think this will help a lot with mid-development debugging/sanity checking

Reply:
Good idea, I like that better